parts/k8s/kubernetesinstalls.sh

```diff
@@ -221,7 +221,10 @@ cleanUpContainerImages() {
     // TODO remove all unused container images at runtime
     docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v ${KUBERNETES_VERSION} | grep 'hyperkube') &
     docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v ${KUBERNETES_VERSION} | grep 'cloud-controller-manager') &
-    docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'hcp-tunnel-front') &
-    docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'kube-svc-redirect') &
-    docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'nginx') &
+    if ! $IS_HOSTED_MASTER; then
```

Comment:
I would change to:

`if [ "$IS_HOSTED_MASTER" = "false" ]; then`

Reply:
done